package.json

```diff
@@ -60,7 +60,7 @@
     "react-inspector": "5.1.1",
     "react-markdown": "4.3.1",
     "react-redux": "7.2.6",
-    "react-router-dom": "4.3.1",
+    "react-router-dom": "4.4.0-beta.6",
```

Comment:
No way we use a beta version here. 

If this is an error in react-router-dom, let's disable this specific warning.

Reply:
upgrade to v5.3.1, (I tried to go to v6.3.0 but it does not work). It's already a good step, to go to v5. :tada:

Reply:
I don't think this is a good idea to have this kind of upgrade within our critical path in order to release the new app. 

We should wait for the release before merging it. (Did you test with harvest & key-libs aka cozy with a vault and without a vault? Did you test all the harvest routes?)

Reply:
you are totally right. Let's wait.